src/Extension/RichModelFormsTypeExtension.php

```diff
@@ -152,6 +152,7 @@ public function configureOptions(OptionsResolver $resolver): void
 
             return $value;
         });
+        $resolver->setDefault('factory_argument', null);
```

Comment:
I think we should also set up the allowed types for the `factory_argument` option (`string` or `null`).